fbw-a32nx/src/wasm/radalt_a320/src/radaltControl.h

```diff
@@ -0,0 +1,617 @@
+		return "EXCEPTION_OBJECT_CONTAINER (34)";
+	case SIMCONNECT_EXCEPTION_OBJECT_AI:					//35
+		return "EXCEPTION_OBJECT_AI (35)";
+	case SIMCONNECT_EXCEPTION_OBJECT_ATC:					//36
+		return "EXCEPTION_OBJECT_ATC (36)";
+	case SIMCONNECT_EXCEPTION_OBJECT_SCHEDULE:				//37
+		return "EXCEPTION_OBJECT_SCHEDULE (37)";
+	default:
+		return "UNKNOWN EXCEPTION";
+	}
+}
+void CALLBACK RadaltDispatchProc(SIMCONNECT_RECV* pData, DWORD cbData, void* pContext)
+{
+	switch (pData->dwID)
+	{
+	case SIMCONNECT_RECV_ID_EVENT:
+	{
+		SIMCONNECT_RECV_EVENT* evt = (SIMCONNECT_RECV_EVENT*)pData;
+
+
+		switch (evt->uEventID)
+		{
+		case EVENT_PAUSED:
+			simPaused = evt->dwData;
+			if (debug) std::cout << "RADALT: [EVENT_PAUSED] = " << simPaused << std::endl;
+			break;
+		case EVENT_4S_TIMER:
+			if (devState == 0) {
+				double acBusState = simVars->getAcBusState1() + simVars->getAcBusState2();
+				double altitudeAGL = simVars->getPlaneAltitudeAGL();
+
+				if (!radarActive && acBusState > 0 && altitudeAGL < MAX_BEAM_DISTANCE)
+				{
+					if (debug) std::cout << "RADALT: [EVENT_4S_TIMER] Probe START ..." << std::endl;
+					getStartupData();
+				}
+
+				if ((radarActive && acBusState < 1) ||
+					(radarActive && acBusState > 0 && altitudeAGL > MAX_BEAM_DISTANCE))
+				{
+					if (debug) std::cout << "RADALT: [EVENT_4S_TIMER] Probe STOP ..." << std::endl;
+					removeProbes();
+				}
+			}
+			break;
+		case EVENT_RADAR_ON:
+			if (debug) std::cout << "RADALT: [EVENT_RADAR_ON]" << std::endl;
+			if (!radarActive)
+			{
+				simVars = new SimVars();
+				getStartupData();
+			}
+			break;
+		case EVENT_RADAR_OFF:
+			if (debug) std::cout << "RADALT: [EVENT_RADAR_OFF]" << std::endl;
+			if (radarActive)
+			{
+				removeProbes();
+			}
+			break;
+		default:
+			std::cout << "RADALT: Unknown event: " << evt->uEventID << std::endl;
+			break;
+		}
+		break;
+	}
+	case SIMCONNECT_RECV_ID_SIMOBJECT_DATA:
+	{
+		SIMCONNECT_RECV_SIMOBJECT_DATA* pObjData = (SIMCONNECT_RECV_SIMOBJECT_DATA*)pData;
+
+		if (pObjData->dwRequestID >= (UINT)REQUEST_PROBE_DATA &&
+			pObjData->dwRequestID < (UINT)REQUEST_PROBE_DATA + MAX_AI) {
+
+			UINT probeIndex = (UINT)pObjData->dwRequestID - (UINT)REQUEST_PROBE_DATA;
+
+			ProbeStruct* pS = (ProbeStruct*)&pObjData->dwData;
+
+			probePosition.altitude = pS->altitude;
+			probePosition.altitudeAboveGround = pS->altitudeAboveGround;
+			probePosition.latitude = pS->latitude;
+			probePosition.longitude = pS->longitude;
+			probePosition.pitch = pS->pitch;            // ground distance
+			probePosition.heading = pS->heading;        // ground bearing
+
+			if (simPaused == false) {
+				updateProbeMesh(probeIndex, preSample, probeBest);
+				readProbeData(probeIndex, probePosition);
+			}
+
+			if (probeIndex == MAX_AI - 1) {
+				if (preSample == 0) {
+					if (simVars->getAcBusState1() == 1 || simVars->getAcBusState2() == 1)
+						simVars->setRadioAltitude(radioAltitude);
+					else
+						simVars->setRadioAltitude(99999);
```

Comment:
```suggestion
					if (simVars->getAcBusState1() == 1 || simVars->getAcBusState2() == 1)
						simVars->setRadioAltitude(radioAltitude);
					else
						simVars->setRadioAltitude(99999);
						
```

enclose with {} for readability please

Reply:
Not sure I understand this suggestion. Do you mean to use {} on the if statement even if its only a one liner? Just asking since I don't see the suggested change